index.ts

```diff
@@ -242,7 +257,7 @@ client.on('message', message => {
     };
 
 
-    if (message.mentions.has(client.user, mentionsOptions)) {
+    if (message.mentions.has(client.user!, mentionsOptions)) {
```

Comment:
What's `!` mean here? This is new to me.

Reply:
Fuck you compiler, I know this property is type `Nullable<T>` but I just want a fucking T